src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawer {
+    protected String color;
+
+    public void printFigure() {
+    }
```

Comment:
There is no need to include method of interface in abstract class if it doesn't contain any implementation